account-kit/signer/src/client/types.ts

```diff
@@ -142,6 +142,7 @@ export type AuthenticatingEventMetadata = {
 
 export type AlchemySignerClientEvents = {
   connected(user: User): void;
+  newUserSignupClient(): void;
```

Comment:
this is super nitpicky... but can we drop the `Client` from the event name? the origin is implied when you subscribe already